code/game/machinery/iv_drip.dm

```diff
@@ -129,14 +133,16 @@
 	if(beaker)
 		// Give blood
 		if(mode)
-			if(beaker.reagents.total_volume)
+			if(beaker.reagents && beaker.reagents.total_volume)
```

Comment:
You can combine these two statements with this operator safely
https://www.byond.com/docs/ref/info.html#/operator/%3f

```suggestion
			if(beaker.reagents?.total_volume)
```